bin/start_sync_stop.py

```diff
@@ -38,7 +38,8 @@
 DEFAULT_SYNC_DURATION = 60
 # Number of seconds between polls of state.
 DEFAULT_POLL_INTERVAL = 5
-SYNC_STATES = ('full', 'validating', 'proposing')
+# The 'proposing' state does not yet have a copy of the ledger.
```

Comment:
By "ledger", are you referring to the ledger agreed upon by the consensus process? the last closed ledger?

Reply:
ok